### ISSUE TITLE
animate().during only once?

### ISSUE THREAD

**Issue by iosonofabio** [wout/svg.js#413]
Congrats for the library, which makes working with SVG a breeze.

What is the clean way of calling a callback during an animation, at a certain point (e.g. 50%)?
I know of `animate().during` but that is called all the time...

Thanks in advance!

**Comment by Fuzzyma** [comment]
There is no other possibility except the one you already mentioned.
`during` gets the position parameter as argument. You can check against this param and call your code when it hits `0.5`. Note that easing is already applied to this position so `0.5` means 0.5 in "space" and not in time

**Comment by iosonofabio** [comment]
Would you consider adding some functionality for this? once-callbacks are a pretty common pattern nowadays...

As an alternative, one could give more info to the callback of `during`, e.g.
```
rect.animate()
  .during(function (pos, lastPos) {
    if ((lastPos < 0.5) & (pos > 0.5))
      // do something...
  })
```

**Comment by Fuzzyma** [comment]
I try to rewrite the whole animation module anyway. I will add this to the list. However this can rly take some time

**Comment by Fuzzyma** [comment]
implemented in 3.2.0 as `once(pos, func, isEased)` (https://github.com/wout/svg.js#once)
